test/unit/lib/monocle_test.js

```diff
@@ -42,6 +42,14 @@ describe('Monocle API Client', function() {
                     this.http.request.calledWith(method.toUpperCase(), '/my-base/foo').should.be.true;
                 });
 
+                it('prepends host if provided', function() {
```

Comment:
it is required ... right now if you don't do setHost the API will return an error